datadog_checks_base/datadog_checks/base/log.py

```diff
@@ -10,6 +10,14 @@
 
 from .utils.common import ensure_bytes
 
+TRACE_LEVEL = 7
```

Comment:
Could you add a comment to detail from where this number come from?

Reply:
5 might make more sense

Reply:
7 is what was set in https://github.com/DataDog/datadog-agent/pull/2826/files#diff-b86e4efc0a6508ff3165b8881d6297b2R156